### DISCUSSION TITLE
Transactions meeting agenda for 2023-02-08

### DISCUSSION THREAD

**Discussion by OBWANDO** [Open-MV/fig-transactions#2] [Meetings]
## Meeting Details

- **Date/Time:** February 8th, 2023 @ Wednesday 1:00 PM ET / 18:00 UTC / 7:00 PM CET
- **Location:** Link should be posted in the `#Transactions` text and voice channel on Discord at least 5 days before the meeting.
- **Moderator:** TBA
- **Note Taker** Person

The [FIG Transactions Meetings](https://github.com/Open-MV/fig-transactions/tree/main/meetings) repo contains the history past meetings, including a link to the agenda, recording, notes, and resources.

## Meeting Agenda
First meeting and operating
FIG charter discussion
Database Feature triage

**Discuss agenda from proposed topics**

## Outcomes from Discussion topics

what should we be doing and what should we not be doing
Need to have people contribute to charter for OMF - how does it drive function reqs

How to define rights and obligations someone should have 
An example was where platform operator has no obligation to support asset ingress/egress, and platform users has no rights to transact or influcence other

Dive deeper into operator obligation - prioritizing platform operator - rights of platform users - obligations of payment service - 

What are bare minimums and obligations of platform operator - 
Need to provide definition of these obligations
Need clear definition of platform operator and put it in the glossary
Need to understand clear platform obligation of payment and users
Need to clearly define transaction in glossary

Need to better define what we are as a FIG that we facilitate transactions, but that we arent the actual trsnaction itself (payment/clearing)
Draw the line from the API and the endpoint provider - 

Need to support the testing of the API for approved plugins -
how do we have right people providing requirements early on - 

---

bill_a123 - transactions ecommerce - 
settlement transactions - should everything be recorded on a ledger
is it the backbone of metaverse protcol - follow ledger of history -

how do we address the ledger as a global or a smaller series of sectional ledgers and others
need a way to query for object identity - 

verify for server - 

## Action Items

Have community start fleshing out the charter
Need to provide and start linking defined terms to ensure we're saying the same thing.

## Items for next week's meeting
**List items that need to be discussed next week**

## Open Discussion Items

List any additional items below!

**Comment by clasoncj** [comment]
Proposed Topic: Requirements gathering from users of this component

* How can we ensure the end users of this component (such as platform operators) are driving requirements?
* Can/should we try to close out fundamental implementation details early before diving into specific feature implementation? 

Example: For transactions involving digital assets, is there consensus from platform operators on the following:

a) Run the ledger that acts as a system of record for digital asset ownership or use a 3rd party
b) Have the ability to directly control the transfer of assets from one user to another
c) Have the ability to prevent the transfer of assets from one user to another
d) Desire to stay completely out of the transfer of digital assets between users but facilitate two users doing a peer to peer transaction

**Comment by will-a123** [comment]
In a similar vein to @clasoncj 's I'm focused on the most abstract interpretation of what a "transaction" should mean:

- At an abstract level, what are the kinds of transactional activities we expect metaverse users to do? Do they look like e-commerce? m-commerce? Traditional commerce? Behavioral recordkeeping? Something else?
- Are there any potentially metaverse-unique activities that we may want to optimize for, like micropayments for access to content or movement between metaverse real-estate?
- Should the fundamental backbone of "the OMF metaverse" actually be the transaction system, with all worlds, entities and objects kept on an OMF ledger?

I think it will also be helpful to put together a list of required transaction types, which was discussed a bit in discord based on this table:

![image](https://user-images.githubusercontent.com/22437489/217439456-ceaf6a70-ae84-4316-a478-b43c01e838e4.png)
